harness/tests/cli/test_user.py

```diff
@@ -33,6 +116,34 @@ def test_user_change_password(mock_getpass: mock.MagicMock) -> None:
         cli.main(["user", "change-password", "tgt-user"])
 
 
+@mock.patch("getpass.getpass")
```

Comment:
Totally totally your call, but suggesting [pytest.mark.parameterize](https://docs.pytest.org/en/latest/how-to/parametrize.html#parametrize-basics) in case you think it makes this easier. I think it's good both ways.